src/main/resources/application.yaml

```diff
@@ -1,6 +1,25 @@
 server:
   port: 5022
 
+spring:
+  jpa:
+    database: POSTGRESQL
+    show-sql: false
+    hibernate:
+      ddl-auto: validate
+      naming:
+        physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
+        implicit-strategy: org.hibernate.boot.model.naming.ImplicitNamingStrategyJpaCompliantImpl
+    properties:
+      dialect: org.hibernate.dialect.PostgreSQLDialect
+      hibernate.format_sql: false
```

Comment:
should be removed as the default value is false

Reply:
Removed